src/views/home/post/Posts.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="post-container" v-if="getPosts">
+    <div v-for="(post, index) in getPosts" :key="index">
+      <v-card class="post">
+        <v-list-item>
+          <v-list-item-content>
+            <v-list-item-title class="headline">{{
+              post.name
```

Comment:
https://www.w3.org/TR/activitystreams-core/#text-representations ->

this would lead to something like ->
```
strip_html_tags(post.name) ? strip_html_tags(post.name) : strip_html_tags(post.summary)
```